src/strategies/spidStrategy.ts

```diff
@@ -43,17 +47,22 @@ export interface IIoSpidStrategy extends Strategy {
  * for spidStrategy object.
  */
 export async function loadFromRemote(
-  idpMetadataUrl: string
+  idpMetadataUrl: string,
+  idpIds: IIdpIds
 ): Promise<{ [key: string]: IDPOption | undefined }> {
```

Comment:
can we return a TaskEither here ?

Reply:
Yes, we can. Had I better change also the related methods in order to avoid the usage of `Promise`s and `async`-`await` for all of them?

Reply:
that would be nice, expecially  for the methods that return Promise<result | error> which is bad : )